packages/template/test/Adaptor.test.js

```diff
@@ -1,10 +1,9 @@
 import { expect } from 'chai';
```

Comment:
This file could use some more comments to explain how the tests are designed - especially around the mock stuff, which is neither trivial nor intuitive